SECURITY.md

```
@@ -58,4 +58,4 @@ For third-party CVEs that may impact Fluent Bit, we will post an assessment and
 
 ---
 
-_Last updated: October 17, 2025_
\ No newline at end of file
+_Last updated: December 18, 2025_
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)